var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -10,16 +10,10 @@ class Sz(CMakePackage):
     """Error-bounded Lossy Compressor for HPC Data"""
 
     homepage = "https://szcompressor.org"
-    url      = "https://github.com/szcompressor/SZ/archive/v2.1.10.tar.gz"
+    url      = "https://github.com/szcompressor/SZ/archive/v2.1.11.tar.gz"
```

Comment:
If you update this URL to `https://github.com/szcompressor/SZ/releases/download/v2.1.11/SZ-2.1.11.tar.gz` then the `url_for_version` shouldn't be needed, right?